pkg/libratoaggregator/aggregator.go

```diff
@@ -0,0 +1,100 @@
+package libratoaggregator
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"sync"
+	"time"
+)
+
+type Gauge struct {
+	Name        string  `json:"name"`
+	MeasureTime int64   `json:"measure_time"`
+	Source      string  `json:"source"`
+	Count       int64   `json:"count"`
+	Sum         float64 `json:"sum"`
+	Min         float64 `json:"min",omitempty`
+	Max         float64 `json:"max",omitempty`
+	SumSquares  float64 `json:"max",omitempty`
```

Comment:
need to question all of these tags - is `omitempty` needed in this context?  Should it be applied to others?